### ISSUE TITLE
Quest: Free your mind

### ISSUE THREAD

**Issue by Myrmidon21** [FeenixServerProject/Wrath_3.3.5_Issue_tracker#138]
Quest given by Baron Silver in the Icecrown Area. The 3 mobs u are supposed to kill and then use the item given by the npc on, never die. They reach a low amount of health (100) and they reset to their starting position. After trying to kill each mob, after reseting they vanish and they reapear after a while. The quest is core in order to continue the quest line of ''Knights of the ebon blade''
Mob names: Vile,Lady Nightswood,The leaper.
Mobs are spotted on The Shadow Vault

**Comment by FRBooo** [comment]
This core quest do work, or at least it did. try clearing your catch.

**Comment by Myrmidon21** [comment]
i have already tried clearing my cache and my WTF. I also tried doing the quest from an another pc but still i had the shame problem. i know that the quest used to work cause i talked with a lot of guys online.

**Comment by execcc** [comment]
Yup, I have done the quest 2 times on other chars already but now it's not working.

**Comment by Goron123** [comment]
I can confirm, been trying it for 2 days at different times and it is not working.  The mobs reset at 1hp then eventually despawn after 3-4 attempts.

**Comment by Goron123** [comment]
Any update on this one?  We can't get the Ebon Blade tabard because of this... :-(

**Comment by marti36** [comment]
Please fix this as soon as you can.  I want to get Ebon Blade rep before Ulduar is released.

**Comment by marti36** [comment]
Any update on it so far?
